persistence/src/test/java/edu/softserve/zoo/persistence/repository/TaskRepositoryTest.java

```diff
@@ -0,0 +1,72 @@
+package edu.softserve.zoo.persistence.repository;
+
+import edu.softserve.zoo.model.Employee;
+import edu.softserve.zoo.model.Task;
+import edu.softserve.zoo.model.TaskStatistics;
+import edu.softserve.zoo.model.ZooZone;
+import edu.softserve.zoo.persistence.config.PersistenceTestConfig;
+import edu.softserve.zoo.persistence.specification.hibernate.impl.GetAllSpecification;
+import edu.softserve.zoo.persistence.specification.hibernate.impl.GetByIdSpecification;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.util.Assert;
+
+import java.time.LocalDateTime;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author Taras Zubrei
+ */
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(classes = PersistenceTestConfig.class)
+@ActiveProfiles("test")
+@Transactional
+public class TaskRepositoryTest {
```

Comment:
you totally break concept of unit testing by doing so. 
there definitely won't be separate task for tests. if you implement functionality, you MUST cover it by tests. 
If you don't use tearDown -> you break unit testing concept -> if you run all the tests together, you will definitely get errors.